From e2e45daa0824d741e58a13cf5101121a71414352 Mon Sep 17 00:00:00 2001
From: Rob Clark <rob@ti.com>
Date: Sun, 22 Nov 2009 13:11:58 -0600
Subject: [PATCH 12/12] fix potential segfault
To: gst_omap4@list.ti.com

don't use the trace macros which dereference port after port has been free'd

Signed-off-by: Rob Clark <rob@ti.com>
---
 omx/gstomx_port.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/omx/gstomx_port.c b/omx/gstomx_port.c
index 6a2fc05..cf41846 100644
--- a/omx/gstomx_port.c
+++ b/omx/gstomx_port.c
@@ -85,7 +85,7 @@ g_omx_port_free (GOmxPort *port)
     g_free (port->buffers);
     g_free (port);
 
-    DEBUG (port, "end");
+    GST_DEBUG ("end");
 }
 
 void
-- 
1.6.3.2

